wowup-electron/src/assets/i18n/de.json

```diff
@@ -7,32 +65,45 @@
     },
     "GET_ADDONS": {
       "CLIENT_TYPE_SELECT_LABEL": "World of Warcraft",
-      "REFRESH_BUTTON": "Aktualisieren",
       "INSTALL_FROM_URL_BUTTON": "Von URL installieren",
+      "REFRESH_BUTTON": "Aktualisieren",
       "SEARCH_LABEL": "Suchen",
       "TABLE": {
         "ADDON_COLUMN_HEADER": "Addon",
         "AUTHOR_COLUMN_HEADER": "Autor",
-        "PROVIDER_COLUMN_HEADER": "Anbieter",
-        "STATUS_COLUMN_HEADER": "Status",
         "DOWNLOAD_COUNT_COLUMN_HEADER": "Anzahl Downloads",
-        "RELEASED_AT_COLUMN_HEADER": "Erscheinungsdatum"
+        "PROVIDER_COLUMN_HEADER": "Anbieter",
+        "RELEASED_AT_COLUMN_HEADER": "Erscheinungsdatum",
+        "STATUS_COLUMN_HEADER": "Status"
```

Comment:
What's causing this file to change the order? While I think it's good to have things sorted, it will cause a lot of merge conflicts when changed.

Is this an automated thing I can run so that my PR will be sorted properly as well? That would solve the merge conflicts

Reply:
looks like you could pull this npm run i18n in and do it